integrations/octopus/main.py

```diff
@@ -0,0 +1,98 @@
+from enum import StrEnum
+from typing import Any, Dict
+from loguru import logger
+from port_ocean.context.ocean import ocean
+from port_ocean.core.ocean_types import ASYNC_GENERATOR_RESYNC_TYPE
+from client import OctopusClient
+
+TRACKED_EVENTS = [
+    "spaces",
+    "projects",
+    "deployments",
+    "releases",
+    "machines",
+]
+
+
+class ObjectKind(StrEnum):
+    SPACE = "space"
+    PROJECT = "project"
+    DEPLOYMENT = "deployment"
+    RELEASE = "release"
+    MACHINE = "machine"
+
+
+async def init_client() -> OctopusClient:
+    client = OctopusClient(
+        ocean.integration_config["octopus_api_key"],
+        ocean.integration_config["server_url"],
+    )
+    return client
+
+
+async def setup_application() -> None:
+    app_host = ocean.integration_config.get("app_host")
```

Comment:
you're using the app_host variable just once so you can access it directly as

if not ocean.integration_config.get("app_host"):
   do something

Reply:
I am using it three times actually. Line 35, 47 and 51